doc/HACKING.md

```diff
@@ -155,7 +158,7 @@ Each address generated after `bip32_max_index` is not included as
 lightning funds.
 
 
-Testing
+Build and Development
```

Comment:
should the `---`s beneath this be expanded to match?

Reply:
Yep, looks better in ASCII :D
Though, a markdown renderer does kill it anyway.